module-2/backend/week_two.md

```diff
@@ -5,37 +5,118 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. At a high level, what is ActiveRecord? What does it do/allow you to do?
+
+  A: ActiveRecord is an ORM that allows you to manipulate a SQL database more easily.
+  
 2. What is a migration?
+
+  A: A migration updates your schema with the changes that you want to make to your database structure.
+    
 3. How does a table relate to a model?
+
+  A: A model represents a table in a database.
+    
 4. What kind of methods are `belongs_to`, and `has_many`? (i.e. class or instance) Give an example.
+
+  A: Class method - e.g. belongs_to :tasks.
```

Comment:
Believe this is technically correct (we might need to reword), but per Casey's lesson today the bigger point that we're trying to make is that you call the methods that these relationships offer you on a specific instance. E.g.:

```ruby
blue_dog   = Dog.find(1) # finds a specific *instance* of a dog.
red_house = blue_dog.house # finds the house associated with the blue_dog
```